src/Parser/Rogue/Assassination/Modules/Spells/EnvenomUptime.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import StatisticBox from "Main/StatisticBox";
+import SpellIcon from "common/SpellIcon";
+import { formatPercentage } from "common/format";
+import STATISTIC_ORDER from 'Main/STATISTIC_ORDER';
```

Comment:
This is up to personal preference, but you can combine the statistic imports into one line like this 
`import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';`